app/assets/scripts/vendor/map.js

```diff
@@ -87,6 +87,36 @@
         setupClickListener('changetype-address', ['address']);
         setupClickListener('changetype-establishment', ['establishment']);
 
+        $.ajax({
+          type: "GET",
+          url: "/get-resource"
+        }).done(function(data){
+            data = JSON.parse(data)
+            for(var i = 0; i < data.length; i++){
+            //we have the pins data, access it here and place pins on map
+              latLng = new google.maps.LatLng(data[i].Latitude, data[i]
+              .Longitude)
+              var marker = new google.maps.Marker({
+                position: latLng,
+                map: map,
+                visible: true,
+                title: data[i].Name
+              });
+
+              var infowindow = new google.maps.InfoWindow({
```

Comment:
If it's easier for you: each marker doesn't have to have a separate infowindow show up when hovered over...you can just have mouseover "alt text" descriptions instead